pycode/memilio-epidata/memilio/epidata/getVaccinationData.py

```diff
@@ -138,12 +144,12 @@ def sanitizing_average_regions(
                 [age_population[dd.EngEng['idCounty']].isin(counties_list)]
                 [[dd.EngEng['idCounty'],
                   age]])[age] / age_population.loc[
-                age_population[dd.EngEng['idCounty']].isin(counties_list)][
-                age].sum()
+                                    age_population[dd.EngEng['idCounty']].isin(counties_list)][
+                                    age].sum()
```

Comment:
We have an automatted formatter. Formatting by hand should by avoided. Not only here.